### ISSUE TITLE
About object categories

### ISSUE THREAD

**Issue by kennymckormick** [joaanna/something_else#4]
Hi, @joaanna . Seems my issue #3 is not solved thus I reopen a new one. I need to know how those placeholders [4k+ unique placeholders] are mapped to your new object categories [total 371 classes in objectsA.json and objectsB.json], which is not provided in the released json file.

**Comment by joaanna** [comment]
Hi @kennymckormick ,

I am not sure I understand your question, so let me explain the procedure here. In our experiments with the objects, we found that they are occurring in many combinations and it would not be feasible to distill two groups of objects that would be completely disjointed. Instead, we selected the frequently appearing objects, those that appear in more than 100 videos and divided them into A and B. The rest of the categories of the objects could be paired with either object from A or B (for example in a video that uses two "something's". 
The issue you described before with the duplicates such as `'rubix cube', 'rubic cube', etc.` was not directly addressed in our work. We got rid of some duplicates by applying some simple preprocessing such as removing 'a' and 'an', 'the' from the object names etc., but we didn't use an exact mapping for those near duplicates. Hope this explains your question.
